docs/clustering.md

```diff
@@ -173,6 +173,39 @@ This also relies on shared storage (such as NFS) being hosted at the following a
 
 Adjust the pre_task `Mount /var/lib/pulp` as needed for your shared storage.
 
+Install the `nfs-utils` package:
+
+```
+# yum install nfs-utils
+```
+Mount the NFS share:
+
+* make sure that the mount point directories exist:
+```
+$ mkdir -p /var/lib/pulp /var/lib/pulp/pulpcore_static
+```
+* use the `showmount` utility to discover the shares from NFS server:
+```
+$ showmount --exports <my-nfs-server-address>
+```
+* open `/etc/fstab` using a text editor, then add the following values:
+```
+<my-nfs-server-address>:<exported-file-system-for-pulp-user-home> /var/lib/pulp nfs defaults,_netdev,nosharecache 0 0
+<my-nfs-server-address>:<exported-file-system-for-webserver-static-dir> /var/lib/pulp/pulpcore_static nfs defaults,_netdev,nosharecache,context="system_u:object_r:httpd_sys_content_rw_t:s0" 0 0
```

Comment:
So they should only set this is if SELinux is enabled (permissive or enforcing). If it is disabled, they should not add the ',context="system_u:object_r:httpd_sys_content_rw_t:s0"'.

This comment applies throughout the page.